src/twisted/conch/ssh/filetransfer.py

```diff
@@ -527,6 +531,21 @@ def connectionMade(self):
         self.sendPacket(FXP_INIT, data)
 
 
+    def connectionLost(self, reason):
+        """
+        Abort any pending requests.
+        """
+
+        # On Python2, we're an oldstyle class, so we can't use super().
+        FileTransferBase.connectionLost(self, reason)
```

Comment:
Even though our `super` implementation is empty, it still exists … I just think it is good practice to invoke the super method when there is one, in case that method gains more functionality in the future. 

However you are right it should update `self.connected`. (I wonder why `Protocol.connectionLost()` doesn't reset `connected` anyway.)